### PR TITLE
Controllers

### PR DESCRIPTION
**Pull request by gdvl777**
Project requirements

- [x] Important: do NOT use rails generate scaffold command. The point of this project is to show you how to set up controllers by yourself.
- [x] Take a look at the wireframes below. Make sure that your app handles the URLs used in the wireframes:

For each URL you should create:

- [x] A route.
- [x] An action in the correct controller.
- [x] A view file.
- [x] View files should include only placeholders written in plain HTML

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -2,5 +2,12 @@
...
 
   # Defines the root path route ("/")
+  get '/users', to: 'users#index'
+
   # root "articles#index"
+  get '/users/:id', to: 'users#show', as: 'user'
+
+  get '/user/:user_id/posts', to: 'posts#index', as: 'user_posts'
+
+  get '/user/:user_id/posts/:id', to: 'posts#show', as: 'user_post'
```
Comment (TedLivist) [inline config/routes.rb]: - [x] The routes on lines 10 and 12 should have `users/`, not `user/`

- [x] **[OPTIONAL]** You can also define these routes using the `resources` method.

  ```
  resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
  ```

**Review by TedLivist — APPROVED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @gdvl777,

Good job so far!
✔️ Implementation of nested routes
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the **[OPTIONAL]** prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: config/routes.rb (hunk 1 above)
Comment (gdvl777) [reply config/routes.rb]: Hey! you approved my PR. What's wrong?

**Review by Gambit142 — APPROVED** [review]
Hi @gdvl777 👋,

## Status: Approved 🟢   

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

### Highlights
- Setup linters properly ✔️ 
- Created routes for the `Users` and `Posts` controllers
- Created view files for the corresponding actions in controllers ✔️ 

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### Please tag me in your question so I can receive the notification. You can tag me by typing @Gambit142

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**